spinalcordtoolbox/reports/qc.py

```diff
@@ -63,21 +64,23 @@ class QcImage(object):
     # _seg_colormap = plt.cm.autumn
 
     def __init__(self, qc_report, interpolation, action_list, stretch_contrast=True,
-                 stretch_contrast_method='contrast_stretching', angle_line=None):
+                 stretch_contrast_method='contrast_stretching', angle_line=None, fps=None):
         """
         :param qc_report: QcReport: The QC report object
         :param interpolation: str: Type of interpolation used in matplotlib
         :param action_list: list: List of functions that generates a specific type of images
         :param stretch_contrast: adjust image so as to improve contrast
         :param stretch_contrast_method: str: {'contrast_stretching', 'equalized'}: Method for stretching contrast
         :param angle_line: float: See generate_qc()
+        :param fps: float: Number of frames per second for output gif images
```

Comment:
Right now, `fps` is only valid for moco, right? It might be good to be explicit about this in the parameter description.